rust/agama-lib/share/profile.schema.json

```diff
@@ -231,6 +231,81 @@
                     }
                   }
                 }
+              },
+              "ieee-8021x": {
```

Comment:
It would be great to describe each element of this section. For instance, `ca_cert` is described but `identity` is not.